central/main.go

```diff
@@ -651,19 +653,29 @@ func customRoutes() (customRoutes []routes.CustomRoute) {
 	})
 
 	scannerDefinitionsRoute := "/api/extensions/scannerdefinitions"
+	// Only grant compression to well-known content types. It should capture files
+	// worth of compression in definition's bundle. Ignore all other types (e.g.,
+	// `.zip` for the bundle itself).
+	definitionsFileGzipHandler, err := gziphandler.GzipHandlerWithOpts(gziphandler.ContentTypes([]string{
+		"application/json",
+		"application/yaml",
+		"text/plain",
```

Comment:
Mime types are not necessarily defined by the filenames suffix, e.g.: YAML and Json files might be rendered as "text/plain" by the implementation, although with Json, considering it is a standard, it's more likely it will not apply.